lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.spec.ts

```diff
@@ -939,6 +896,13 @@ describe('StartProcessCloudComponent', () => {
 
             expect(card).toBeTruthy();
         });
+
+        it('should emit value when process started', () => {
+            component.success.pipe(first()).subscribe((data: ProcessInstanceCloud) => {
+                expect(data).not.toBeNull();
```

Comment:
please let's be more specific pass a define data and expect a defined data. 

Again, in my opinion we could merge some different unit into 1

Fill out the start form / press submit / expect the component emits the correct payload